backend/CircleForms/Models/MongoDbRange.cs

```diff
@@ -1,14 +1,13 @@
-﻿namespace CircleForms.Models
+﻿namespace CircleForms.Models;
+
+public class MongoDbRange
```

Comment:
Change this and other similar classes into `record`s?

Reply:
records due to `{ init; }` can break serialization and it's not required for range to be natively comparable